### DISCUSSION TITLE
[react] <search> element for react types

### DISCUSSION THREAD

**Discussion by nikeee** [DefinitelyTyped/DefinitelyTyped#65402] [Issues with a @types package] [answered]
HTML has landed a `<search>` Element in March: https://html.spec.whatwg.org/multipage/grouping-content.html#the-search-element (more here[^1]).

It doesn't seem supported in the current React types.

Is this in scope? If so, is this open for contribution?

[^1]: https://www.scottohara.me/blog/2023/03/24/search-element.html

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
Looks like element was implemented in webkit - https://github.com/WebKit/WebKit/pull/13887

**Comment by eps1lon** [comment]
Makes sense, feel free to add it

**Reply by nikeee** [reply, marked as answer]
Will do!

**Comment by Semigradsky** [comment]
Already implemented
